server/package.json

```diff
@@ -15,7 +15,8 @@
     "eslint-config-skyscanner": "^3.0.0",
     "eslint-plugin-import": "^2.11.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.12.4"
+    "eslint-plugin-react": "^7.12.4",
+    "jest": "^24.7.1"
```

Comment:
I'm not sure if there are any major differences but just noticing that BRS uses jest `23.6.0` so we would have version inconsistencies between client and server

https://github.com/Skyscanner/backpack-react-scripts/blob/fork/packages/react-scripts/package.json#L43

Reply:
Good spot - I'll align this to BRS 7 to reduce the burden of installing. Hopefully it'll cache the first time and avoid downloading the second.